Automattic/wp-calypso
# Checkout: Move payment method translation functions out of types

#### Changes proposed in this Pull Request

There are a handful of functions in new checkout that deal with the translation of payment method names from their "long" (eg: `WPCOM_Billing_Ebanx_Redirect_Brazil_Tef`) forms to their "short" forms (eg: `brazil-tef`) and related transformations. Currently these functions live under the `types/` directory, but as they are not types, that may be misleading. This PR moves them into the `lib/` directory of new checkout into a single new file.

#### Testing instructions

Load checkout, proceed to the last step and verify that there are payment methods listed.
